tests/test_callback_for.py

```diff
@@ -0,0 +1,119 @@
+import scrapy
+import pytest
+from scrapy.utils.reqser import request_to_dict
+
+from scrapy_po.webpage import ItemPage, ItemWebPage
+from scrapy_po.utils import (
+    callback_for,
+    is_response_going_to_be_used,
+    DummyResponse,
+)
+
+
+class FakeItemPage(ItemPage):
+
+    def to_item(self):
+        return 'it works!'
+
+
+class FakeItemWebPage(ItemWebPage):
+
+    def to_item(self):
+        return 'it works!'
```

Comment:
Probably having different texts here can avoid unwanted changes.
Or missing an error because of some wrong type mapping

Reply:
Done